mongodb-advanced/components/people-service.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+export default function factory($http, brAlertService) {
+  const service = {};
+
+  service.savePerson = (person, callback) => {
+    Promise.resolve($http.post('/people/', person))
```

Comment:
This is a pattern that is found in a lot of angular things, but the new way of doing this is *not* to wrap $http in a `Promise`.  So, you can just
```
return $http.post(...);
```
or even
```
service.foo = () => $http.post(...);
```